src/crimson/os/seastore/cache.cc

```diff
@@ -1192,7 +1192,8 @@ CachedExtentRef Cache::duplicate_for_write(
     // deepcopy the buffer of exist clean extent beacuse it shares
     // buffer with original clean extent.
     auto bp = i->get_bptr();
-    auto nbp = ceph::bufferptr(bp.c_str(), bp.length());
+    auto nbp = ceph::bufferptr(buffer::create_page_aligned(bp.length()));
```

Comment:
out of scope, can you help me understand why is this preferred? There are other users which aren't ensuring this